include/Contact.php

```diff
@@ -306,9 +306,10 @@ function get_contact_details_by_url($url, $uid = -1, $default = array()) {
 		$profile["micro"] = $profile["thumb"];
 	}
 
-	if ((($profile["addr"] == "") || ($profile["name"] == "")) && ($profile["gid"] != 0) &&
-		in_array($profile["network"], array(NETWORK_DFRN, NETWORK_DIASPORA, NETWORK_OSTATUS))) {
-		Worker::add(PRIORITY_LOW, "update_gcontact", $profile["gid"]);
+	if ((($profile["addr"] == "") || ($profile["name"] == "")) && ($profile["gid"] != 0)
+		&& in_array($profile["network"], array(NETWORK_DFRN, NETWORK_DIASPORA, NETWORK_OSTATUS))
+	) {
+		Worker::add(PRIORITY_LOW, "update", $profile["gid"]);
```

Comment:
This shouldn't be changed as it references the cronjob in `include/update_gcontact.php`.